### PR TITLE
feat: Remove solana support

### PR DESCRIPTION
**Pull request by maximgeerinck**

### CODE REVIEW

**Review by viet10101988 — APPROVED** [review]
we do version bump in next branch right?

**Comment by maximgeerinck** [comment]
> we do version bump in next branch right?

i already bumped it, but will merge on monday so nothing's blocking

**Review by mihaiberq — APPROVED** [review]
RIP

File: src/transaction/Transaction.ts [hunk 1]
```diff
@@ -1,9 +1,7 @@
-import { Commitment } from "@solana/web3.js";
-
 export interface Transaction {
   txId: string;
   timestamp?: number;
   success: boolean;
   // This function waits until the transaction has been mined
-  wait: (confirmations?: number | string | Commitment) => Promise<Transaction>;
+  wait: (confirmations?: number | string) => Promise<Transaction>;
```
Comment (sneljo1) [inline src/transaction/Transaction.ts:6]: string can also be removed

File: src/utils/connection.ts [hunk 2]
```diff
@@ -1,8 +1,7 @@
-import { Connection } from "@solana/web3.js";
 import { getChainByNetworkId } from "../config/chains";
 import { Chain } from "../interfaces/Network";
 
 export const getConnectionByChainId = (chainId: Chain) => {
```
Comment (sneljo1) [inline src/utils/connection.ts:4]: Can be fully removed, this function is specific to fetching the connection, if we no longer fetch it, we no longer need it

File: tests/e2e/erc1155.e2e.spec.ts [hunk 3]
```diff
@@ -186,8 +185,6 @@ describe("ERC1155 - E2E", () => {
 
       const nft = await createNft(refinable, 100);
 
-      await sleep(1000);
```
Comment (sneljo1) [inline tests/e2e/erc1155.e2e.spec.ts:189]: Can we not keep this? I believe it was there for a reason

**Review by sneljo1 — APPROVED** [review]
Couple small remarks

File: src/utils/connection.ts (hunk 2 above)
Comment (maximgeerinck) [reply src/utils/connection.ts:4]: ok

**Pull request merged**